### PR TITLE
Setup and controllers

### PR DESCRIPTION
**Pull request by elmar8287**
- Create a new app. Give it a meaningful name.
- Initialize the project with Git.
- Make sure that the project has Postgres database set up.
- Take a look at the wireframes below. Make sure that your app handles the required URLs.
- For each URL you should create:

1. A route.
2. Action in the correct controller.
3. A view file.

- View files should include only placeholders written in plain HTML, e.g:

1. Here is a list of posts for a given user.

- Make sure that you added code to handle only the 4 required URLs.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,3 +1,9 @@
+# frozen_string_literal: true
+
 Rails.application.routes.draw do
+  get '/users/:id/posts', to: 'posts#index'
+  get '/users/:id/posts/:id', to: 'posts#show'
+  get '/users', to: 'users#index'
+  get '/users/:id', to: 'users#show'
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
 end
```
Comment (bini-i) [inline config/routes.rb:9]: - [ ] [**_Optional_**] Good job implementing only 4 of the required paths. But I just wanted to mention that using rails `resources` DSL by defaults adds named path helpers that you can later use to refer to different routes easily. See the difference below:

Screenshot from 2021-12-11 09-49-44

**Review by bini-i — APPROVED** [review]
# Status: Approved :heavy_check_mark:          
### Hello @elmar8287  :wave: 

**Congratulations**, your project meets all the requirements and it is **approved** :tada:.

#### To highlight
- Correct linters setup and Gitflow usage.
- Database adapter correctly setup.
- all the necessary routes are implemented.

#### If you have any questions feel free to ask within the PR thread.
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
### HAPPY CODING :computer: :blush: 

> #### Reviewed by Binyam Hailemeskel (Code Reviewer)
> GitHub: [@bini-i](https://github.com/bini-i) 
> Twitter: [@binyamshewa](https://twitter.com/binyamshewa)
> LinkedIn: [LinkedIn](https://www.linkedin.com/in/binyam-hailemeskel-728048151/)

File: config/routes.rb (hunk 1 above)
Comment (elmar8287) [reply config/routes.rb:9]: Thanks for the comment

**Comment by elmar8287** [comment]
Thanks for the review!

**Pull request merged**